### PR TITLE
Add CLI command reference docs page; update API Services docs page; add create/fetch docs page with more sample files and examples

### PR DESCRIPTION
**Pull request by rsavajiyani**
https://deploy-preview-1546--axway-open-docs.netlify.app/docs/central/cli_central/cli_apiservices/

https://deploy-preview-1546--axway-open-docs.netlify.app/docs/central/cli_central/cli_command_reference

https://deploy-preview-1546--axway-open-docs.netlify.app//docs/central/cli_central/cli_create_fetch_resources

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

-Added a CLI command reference page that will serve as an 'index' for CLI users
-Updated the docs page centered around APIServices and the CLI with sample output
-Added a CLI create/fetch resource page with examples and links to sample resource files the user can use

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [X] You have verified that all status checks have passed

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by rsavajiyani** [comment]
Hi @andreamussap , I'd appreciate your feedback on these additions/modifications to the CLI docs.

CC @robertengaxway

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+Prints a table of the most important information about the specified resources.
+
+If the desired resource type is scoped you will need to specify the scope name by providing `-s, --scope <name>` flag.
+
+You can query for more than one resource if using comma-separated resources in the command (see the examples).
+
+Querying for multiple resources will display multiple result tables, one result table for each resource you fetch.
+
+To see the list of all available resources on the server run `amplify central get`
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:21]: Prefer not to mention "server".
```suggestion
To see the list of all available resources from AMPLIFY Central, run `amplify central get`
```

**Comment by andreamussap** [comment]
@rsavajiyani @robertengaxway I've made changes to **section GET** to improve readability:

https://deploy-preview-1546--axway-open-docs.netlify.app/docs/central/cli_central/cli_command_reference/

- Removed the heading3 (###) because they create unnecessary repetition on the right-side menu. 
- Reorganized section Example. All examples and commands together in one codeblock was confusing.
- Placed usage, options, and arguments in a table. It seems to me that they read better now, but let me know your thoughts about these changes before I can continue.

Thanks.

**Comment by andreamussap** [comment]
> @rsavajiyani @robertengaxway I've made changes to **section GET** to improve readability:
> 
@robertengaxway I'm waiting for your feedback.

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 2]
```diff
@@ -160,7 +158,7 @@ name by providing `-s, --scope <name>` flag.
 
 To see the list of all available resources on the server run `amplify central delete`.
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:174]: replace "server" with "API Server" as used in https://docs.axway.com/bundle/axway-open-docs/page/docs/central/integrate_with_webhooks/index.html 
We can use the formal "Axway API Server" in the intro section of the Command Summary or the Central CLI intro.

Comment (rsavajiyani) [reply content/en/docs/central/cli_central/cli_command_reference.md]: @robertengaxway I made the suggested change on that line(server --> API Server), thanks

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+weight: 130
+date: 2021-01-13T00:00:00.000Z
+description: Learn about the different AMPLIFY Central CLI commands you can use.
+---
+
+This section contains the basic commands for creating, fetching, updating, and deleting various Axway API Server assets using the Amplify Central CLI. Each command is followed by a brief description of what the command does, an explanation of the proper command syntax including command arguments and options, along with example syntax for various use-cases. Use this page to quickly lookup more information about the Amplify Central CLI command you're utilizing.
+
+## get
+
+This command lists one or more resources. It also prints a table of the most important information about an specified resources. If the desired resource type is scoped, you will need to specify the scope name by providing the `-s, --scope <name>` flag.
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:13]: Should we explain the meaning of a scoped resource versus resources that are not scoped with examples?

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+amplify central get envs
+```
+
+To get all environments in yaml format
+
+```
+amplify central get environments -o yaml
+```
+
+To get environment by Resource/Common name in json format
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:50]: To make it relevant to the example provided.    Suggest saying edit to say  "Resource/Common name (i.e. myenv)"

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+amplify central get webh,apis
+```
+
+To get all environments and API services
+
+```
+amplify central get envs,apisvc
+```
+
+To get an environment and an apiservice which matches a Resource/Common Name in a specified scope (scope is required after -s flag) in json format
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:80]: Suggest edit to say "(scope is env1 in the example and required after -s flag)"

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+
+### Install Options
+
+`--client-id=<value>`   Override your DevOps account's client ID
+
+`--region=<value>`      Override your region config
+
+#### **install: agents configuration**
+
+Install API Gateway v7 / Amazon API Gateway / Kubernetes agents
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:274]: I asked Cyrille if this needs to include new parameter for 'azure' agents in the CLI.  Meaning has it been done yet.

Comment (robertengaxway) [reply content/en/docs/central/cli_central/cli_command_reference.md:274]: No need to change to mention Azure yet.   Its not a support optional parameter yet.

Comment (robertengaxway) [reply content/en/docs/central/cli_central/cli_command_reference.md:274]: Correction, the install agents will support Azure this week.
image

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 7]
```diff
@@ -0,0 +1,295 @@
...
+description: Learn about the different AMPLIFY Central CLI commands you can use.
+---
+
+This section contains the basic commands for creating, fetching, updating, and deleting various Axway API Server assets using the Amplify Central CLI. Each command is followed by a brief description of what the command does, an explanation of the proper command syntax including command arguments and options, along with example syntax for various use-cases. Use this page to quickly lookup more information about the Amplify Central CLI command you're utilizing.
+
+## get
+
+This command lists one or more resources. It also prints a table of the most important information about an specified resources. 
+
+Resources can be *scoped* or *unscoped*. A resource's scope refers to the accessibility of the resource. A scoped resource type will be deleteed when its scoped/related resource is deleted. Access to that resource type is controlled by authorization checks on the scope. An *unscoped* resource's availability/accessibility is *independent* of the accessibility of other resources.
```
Comment (rsavajiyani) [inline content/en/docs/central/cli_central/cli_command_reference.md:15]: @robertengaxway I added this explanation of scoped vs unscoped. Let me know if it's ok or if I should revise it.

File: content/en/docs/central/cli_central/cli_command_reference.md [hunk 8]
```diff
@@ -10,9 +10,13 @@ This section contains the basic commands for creating, fetching, updating, and d
 
 ## get
 
-This command lists one or more resources. It also prints a table of the most important information about an specified resources. If the desired resource type is scoped, you will need to specify the scope name by providing the `-s, --scope <name>` flag.
+This command lists one or more resources. It also prints a table of the most important information about an specified resources. 
 
-You can search for more than one resource if you use comma-separated resources in the command. The search for multiple resources will display multiple result tables, one result table for each resource you fetch. To see the list of all available resources from AMPLIFY Central, run:
+Resources can be *scoped* or *unscoped*. A resource's scope refers to the accessibility of the resource. A scoped resource type will be deleteed when its scoped/related resource is deleted. Access to that resource type is controlled by authorization checks on the scope. An *unscoped* resource's availability/accessibility is *independent* of the accessibility of other resources.
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_command_reference.md:15]: typo on 'deleteed', should be 'deleted'

**Review by robertengaxway — APPROVED** [review]
Nice work Ravi, I approve of the latest changes

**Comment by rsavajiyani** [comment]
@andreamussap Thanks for your help and revisions. I implemented @robertengaxway 's suggestions and he said it's ok to merge from his side as well. Let me know if there's anything else needed from me before it's ready for you to merge :).

**Comment by andreamussap** [comment]
Review of cli_command_reference.md is done. 

Still need to review cli_apiservices.md and cli_create_fetch_resources.md, but due to APIM release notes on Jan 30th, I need to prioritize APIM pull requests first. I'll come back to this review ASAP.

**Review by andreamussap — APPROVED** [review]

**Pull request merged**